setup.cfg

```diff
@@ -16,11 +16,17 @@ universal = 1
 
 [flake8]
 exclude = docs
-max-line-length = 120
+max-line-length = 88
```

Comment:
SAD FACE

Reply:
See: https://github.com/psf/black#line-length
And: the "minimal, black-compatible flake8" lines in that README

Reply:
Wow use 88 or else ;)

Reply:
Watch out. The big bad CI runner will getcha :fearful:

Reply:
cellbrowser-tools uses 
```[flake8]
exclude = docs
max-line-length = 120
ignore = E203, E266, E501, W503
max-complexity = 18
select = B,C,E,F,W,T4,B9```

Reply:
In the year 2020 and beyond, I will ride or die with max-line-length=120